Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/packages.config

```diff
@@ -1,7 +1,55 @@
+  <package id="System.IO.Compression.ZipFile" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.IO.FileSystem" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.IO.FileSystem.Primitives" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Linq" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Linq.Expressions" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Net.Http" version="4.3.1" targetFramework="monoandroid71" />
+  <package id="System.Net.Primitives" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Net.Sockets" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.ObjectModel" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Reflection" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Reflection.Extensions" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime.Handles" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime.InteropServices" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Runtime.Numerics" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Text.Encoding" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Text.Encoding.Extensions" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Text.RegularExpressions" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Threading" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Threading.Tasks" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Threading.Timer" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="monoandroid71" />
+  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="monoandroid71" />
```

Comment:
Not directly.. but yes. I'm using an Autofac version that is targeting .NET Standard, and this is what you get when referencing .NET standard package (it's distributed via Nuget).